flowershop/models/flower_water.py

```diff
@@ -0,0 +1,20 @@
+from odoo import fields,models
+
+
+class flowerwater(models.Model):
+    
+    _name = "flower_water"
```

Comment:
```suggestion
    _name = "flower.water"
```

odoo model naming convention separates words by "."